### ISSUE TITLE
MigrateMakeCommand::getPath() does not exist

### ISSUE THREAD

**Issue by tobz-nz** [ahuggins/open-on-make#28]
I've been getting this error since last update (not sure what version I was on previously)
it only happens on some make commands, not all. `make:migration` is one that it does happen on.

```
Method Illuminate\Database\Console\Migrations\MigrateMakeCommand::getPath() does not exist
```

it happens in `/vendor/ahuggins/open-on-make/src/Listeners/OpenOnMake.php:139`
```
    137|     public function determineFilePath()
    138|     {
  > 139|         $pathMethod = new \ReflectionMethod($this->getCommandClass(), 'getPath');
    140|         $pathMethod->setAccessible(true);
    141| 
    142|         $qualifyMethod = new \ReflectionMethod($this->getCommandClass(), 'qualifyClass');
    143|         $qualifyMethod->setAccessible(true);
```

I've not had a chance to dig into it.

**Comment by ahuggins** [comment]
Cool thanks, I’ll check it out when I get a chance

**Comment by ahuggins** [comment]
New version should fix this. Issue was the MigrateMakeCommand does not extend the GeneratorCommand class like pretty much all the other ones do. So I had to work around that as a special case. I also saw that the View package does not extend this either, and really anything that does not...will now use Finder to search for the `name` that is passed in to any command within the base_path() directory. It will automatically add .php to the `name` and search.
